src/main/python/ayab/about.py

```diff
@@ -17,15 +17,19 @@
 #    Copyright 2014 Sebastian Oliva, Christian Obersteiner, Andreas Müller, Christian Gerbrandt
 #    https://github.com/AllYarnsAreBeautiful/ayab-desktop
 
+from __future__ import annotations
 from PySide6.QtWidgets import QFrame
 from PySide6.QtCore import Qt, QCoreApplication
 
 from .about_gui import Ui_AboutForm
 from . import utils
-
+from . import ayab_logo_rc
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from .ayab import GuiMain
```

Comment:
We need to do this little dance to avoid a circular import error at runtime- luckily we can fence the annotations!